ee_plugin/utils.py

```diff
@@ -401,3 +403,59 @@ def download_tile(
     with open(out_path, "wb") as f:
         f.write(response.content)
     logger.debug(f"Tile saved to {out_path}")
+
+
+def get_ee_properties(asset_id: str) -> Optional[List[str]]:
+    """
+    Get property names from any Earth Engine asset.
+    """
+    # TODO: could caller of the function provide a logger?
```

Comment:
@alukach do you have any ideas here? Do we need to provide the logger as a parameter to the function to let the caller (i.e. our image collection dialog) to define how to log things? Not sure if there's more clever ways...

Reply:
fixed by using a `silent` paremeter!